WootricSDK/WootricSDK/Wootric.h

```diff
@@ -77,4 +78,9 @@
 
 + (void)setCustomValueForResurveyThrottle:(NSNumber *)resurveyThrottle visitorPercentage:(NSNumber *)visitorPercentage registeredPercentage:(NSNumber *)registeredPercentage dailyResponseCap:(NSNumber *)dailyResponseCap;
 
++ (void)setSendButtonBackgroundColor:(UIColor *)color;
++ (void)setNoThanksColor:(UIColor *)color;
++ (void)setSliderColor:(UIColor *)color;
++ (void)setSocialSharingColor:(UIColor *)color;
```

Comment:
I would say it’s ok to trust the user since it will appear on their application.